obal/data/modules/srpm.py

```diff
@@ -0,0 +1,122 @@
+"""
+Build SRPM
+"""
+
+import shutil
+import os
+import subprocess
+from tempfile import mkdtemp
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.obal import chdir # pylint:disable=import-error,no-name-in-module
+
+def run_command(command):
+    """
+    Run a system command
+    """
+    return subprocess.check_output(
+        ' '.join(command),
+        universal_newlines=True,
+        shell=True,
+        stderr=subprocess.STDOUT
+    )
+
+
+def copy_sources(spec_file, package_dir, sources_dir):
+    """
+    Copy RPM sources to rpmbuild environment
+    """
+    command = ["spectool", "--list-files", spec_file]
+    sources = run_command(command)
+
+    for source in sources.split("\n"):
+        if not source:
+            continue
+
+        source_link = source.split(' ')[1]
+
+        if not source_link.startswith('http'):
+            shutil.copy(os.path.join(package_dir, source_link), sources_dir)
+
+    with chdir(package_dir):
+        run_command(["git-annex", "lock"])
+        annexed_files = run_command(["git-annex", "find", "--include='*'"])
+        annexed_files = annexed_files.splitlines()
+
+        if not annexed_files:
+            return
+
+        run_command(["git-annex", "get"])
+
+        try:
+            run_command(["git-annex", "unlock"])
+        finally:
+            for annex in annexed_files:
+                shutil.copy(annex, sources_dir)
+
+            run_command(["git-annex", "lock", "--force"])
+
+
+def main():
+    """
+    Build a package using tito
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            package=dict(type='str', required=False),
+            scl=dict(type='str', required=False),
+            output=dict(type='path', required=False),
+        )
+    )
+
+    package = module.params['package']
+    output = module.params['output']
+    scl = module.params['scl']
+
+    spec_file = os.path.join(package, '%s.spec' % os.path.basename(package))
+
+    try:
+        base_dir = mkdtemp()
+        sources_dir = os.path.join(base_dir, 'SOURCES')
+        build_dir = os.path.join(base_dir, 'BUILD')
+
+        os.mkdir(sources_dir)
+        os.mkdir(build_dir)
+
+        copy_sources(spec_file, package, sources_dir)
+        shutil.copy(spec_file, base_dir)
+
+        command = ['rpmbuild', '-bs']
+        command += ['--define', '"_topdir %s"' % base_dir]
+        command += ['--define', '"_sourcedir %s"' % sources_dir]
+        command += ['--define', '"_builddir %s"' % build_dir]
+        command += ['--define', '"_srcrpmdir %s"' % base_dir]
+        command += ['--define', '"_rpmdir %s"' % base_dir]
```

Comment:
When you drop the join (https://github.com/theforeman/obal/pull/232/files#r468449779) I think these are double quoted, which is wrong. It's the same as https://github.com/theforeman/puppet-foreman_proxy_content/pull/274#discussion_r454312531
```suggestion
        command += ['--define', '_topdir %s' % base_dir]
        command += ['--define', '_sourcedir %s' % sources_dir]
        command += ['--define', '_builddir %s' % build_dir]
        command += ['--define', '_srcrpmdir %s' % base_dir]
        command += ['--define', '_rpmdir %s' % base_dir]
```